doc/release-notes.md

```diff
@@ -36,49 +36,429 @@ reindex or re-sync the whole chain.
-while processing notifications for every tx. Running `keypoolrefill` with a
-large number of keys will no longer lockup the GUI and can be interrupted.
-Running `upgradetohd` can also be interrupted now.
+## Testnet Breaking Changes
 
-## Changes in RPCs, commands and config options
+A new testnet only LLMQ has been added. This LLMQ is of the type `LLMQ_25_67`,
+this LLMQ is only active on testnet. This LLMQ does not remove the `LLMQ_100_67`
+from testnet, however that quorum will perform no role anymore. See the [LLMQ
+types](https://github.com/dashpay/dips/blob/master/dip-0006/llmq-types.md) for
+specific parameters of the LLMQ. This LLMQ became active on testnet at the
+height of 847000.
 
-- `wipewallettxes`: New RPC command which removes all wallet transactions
-- `wipetxes`: New command for `dash-wallet` that removes all wallet transactions
-- `masternodelist`: New mode `hpmn` filters only HPMNs/EvoNodes
-- `protx list`: New type `hpmn` filters only HPMNs/EvoNodes
-- `-blockversion` config option is allowed on non-mainnet networks now
+__TODO__: fixed subsidy base after `v20`
 
-## Other changes
+## Sentinel deprecation
 
-There were a few other minor changes too, specifically:
-- Added Kittywhiskers Van Gogh (kittywhiskers) and Odysseas Gabrielides
-(ogabrielides) to contributors list in 19.2.0 release notes
-- There should be no false "unknown rules activated" warning in GUI and RPCs now
-- Empty `settings.json` file no longer results in node startup failure
-- Block processing was slightly optimized
-- BLS library was updated to version 1.3.0 to fix a couple tiny issues
-- Fixed a couple of small issues in tests
+Sentinel functionality has been integrated directly into v20.0.0, so it will no
+longer be necessary for masternodes to run the standalone Sentinel application.
+During the Superblock Maturity Window (1662 blocks), payee Masternodes will try
+to create, sign and submit a Superblock Candidate. Several RPC commands have
+been updated to prevent conflicts between v20.0.0 and existing Sentinel
+installs. It is recommended to remove or disable Sentinel after updating
+masternodes to v20.0.0.
 
-# v19.3.0 Change log
+## P2P and network changes
 
-See detailed [set of changes](https://github.com/dashpay/dash/compare/v19.2.0...dashpay:v19.3.0).
+### Removal of reject network messages from Dash Core (BIP61)
+
+This feature has been disabled by default since Dash Core version 19.0.0.
+Nodes on the network can not generally be trusted to send valid ("reject")
+messages, so this should only ever be used when connected to a trusted node.
+
+Since Dash Core version 20.0.0 there are extra changes:
+
+The removal of BIP61 `REJECT` message support also has the following minor RPC
+and logging implications:
+
+- `testmempoolaccept` and `sendrawtransaction` no longer return the P2P `REJECT`
+  code when a transaction is not accepted to the mempool. They still return the
+  verbal reject reason.
```

Comment:
Not sure what is meant by "verbal reject reason" :thinking: It returns a message but no error code?

Reply:
yes, that's a copy-paste from `doc/release-notes-17004.md`